app/src/main/res/layout/course_section_item.xml

```diff
@@ -1,73 +1,99 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+
+        android:layout_marginStart="8dp"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_status_closed" />
 
-    <LinearLayout
-        android:layout_width="match_parent"
+
+    <ImageView
+        android:id="@+id/trackingLogo"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_weight="1"
-        android:orientation="horizontal">
 
-        <ImageView
-            android:id="@+id/section_status"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="18dp"
-            android:layout_weight=".01"
-            android:paddingStart="14dp"
-            android:paddingEnd="14dp"
-            android:paddingBottom="4dp"
-            app:srcCompat="@drawable/ic_status_closed" />
+        android:layout_marginStart="40dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/section_title"
+        app:srcCompat="@drawable/ic_tracking_ppl_logo" />
 
-        <TextView
-            android:id="@+id/section_title"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="14dp"
-            android:layout_weight="0.8"
-            android:ellipsize="end"
-            android:fontFamily="sans-serif-medium"
-            android:gravity="bottom"
-            android:maxLines="2"
-            android:scrollHorizontally="true"
-            android:text="LEC 001 / W 7:30PM"
-            android:textColor="@android:color/black"
-            android:textSize="16sp" />
+    <TextView
+        android:id="@+id/section_title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="32dp"
+        android:layout_marginBottom="16dp"
+        android:layout_weight="0.8"
+        android:ellipsize="end"
+        android:fontFamily="sans-serif-medium"
+        android:gravity="bottom"
+        android:maxLines="2"
+        android:scrollHorizontally="true"
+        android:text="LEC 001 / W 7:30PM"
+        android:textColor="@android:color/black"
+        android:textSize="16sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/button_track"
+        app:layout_constraintStart_toEndOf="@+id/section_status"
+        app:layout_constraintTop_toTopOf="parent" />
 
-        <Button
-            android:id="@+id/button_remove"
-            android:layout_width="90dp"
```

Comment:
why is this hardcoded?

Reply:
That was how they were originally defined but I can change them to wrap content